### DISCUSSION TITLE
How to add a "create child note" botton in tools bar?

### DISCUSSION THREAD

**Discussion by yu-jingrui** [TriliumNext/Trilium#6055] [Ideas] [answered]
How can I add a "create child note" botton in tools bar like this:
![image](https://github.com/user-attachments/assets/16441a80-1bcd-410f-9e2e-b20c045a085f)

I find it really annoying to look for the "plus" botton in the tree view.

**Comment by eliandoran** [comment, marked as answer]
@yu-jingrui , the short answer is that it's not currently possible to add buttons to the ribbon, only to the launch bar (the one on the left side of the screen).

The long answer is that it's theoretically possible with a custom script, which adds a new widget and floats it near the other buttons.

Are you looking explicitly for a button to create child note? There is already a keyboard combination which already does the same thing (Ctrl+P).

**Reply by yu-jingrui** [reply]
Thanks. I know about the keyboard combination. It's just easier to remember less key bindings in life. I guess I will try to add it as a widget.
